tools/zxp/generate_zxp.ps1

```diff
@@ -19,4 +93,17 @@ foreach ($CURR_HOST in $HOSTS) {
 
     # Generate and sign the ZXP file with the OpenPype certificate
     & $PATH_ZXP_SIGN_SOFTWARE -sign $HOST_ZXP_SOURCE $HOST_ZXP_DEST $PATH_ZXP_CERTIFICATE OpenPype
```

Comment:
You need to generate the ZXP after bumping the version not before, else you will have the wrong version number on the generated ZXP.
Move this operation/line at the end after the bump of the version and the new checksum generation

Reply:
Effectivement, je corrige ça